### ISSUE TITLE
.log ignores options if not used with a callback

### ISSUE THREAD

**Issue by Darkhogg** [steveukx/git-js#469]
if using the `.log` method with promises (i.e., without passing a callback), the options object is completely ignored, resulting in logs always including every commit instead of the intended range (among other things)

```
git.log({ from: 'HEAD~10', to: 'HEAD' }).then(log => console.log(log.all.length));
// prints: 697 (the total amount of commits in my repo)

git.log({ from: 'HEAD~10', to: 'HEAD' }, (err, log) => console.log(log.all.length));
// prints: 10 (expected result)
```

**Comment by Darkhogg** [comment]
Woops, I was not using the recommended `simple-git/promise` import, ignore me.

**Comment by steveukx** [comment]
Hi, thanks for opening the issue... you actually should be able to import from just `simple-git` (ie: without the trailing `/promise`), `git.log` had an issue in `2.7.0` which has just been resolved in `2.7.1`. 

If you have further problems, please do open another issue.
